### PR TITLE
Update avidemux_ko.ts

### PR DESCRIPTION
**Pull request by sheppaul**
Fixed some translations

### CODE REVIEW

File: avidemux/qt4/i18n/avidemux_ko.ts [hunk 1]
```diff
@@ -1940,7 +1940,7 @@ Drop timing informations ?</source>
     <message>
         <location line="+16"/>
         <source>Pr&amp;eferences</source>
-        <translation>환경 설정(&amp;E)</translation>
+        <translation>환경 설정</translation>
```
Comment (eumagga0x2a) [inline avidemux/qt4/i18n/avidemux_ko.ts]: Why do you want to remove the kbd accelerator? Does it look confusing? It works fine for me at least on Linux with a usual pc105 keyboard type in 한글 mode. Does it fail for you?

All other changes are OK for me, thank you.

File: avidemux/qt4/i18n/avidemux_ko.ts [hunk 2]
```diff
@@ -945,11 +945,11 @@ Drop timing informations ?</source>
     <message>
         <location line="+10"/>
         <source>© 2001 - 2017  Mean</source>
-        <translation>© 2001 - 2017  Mean</translation>
+        <translation>© 2001 - 2018  Mean</translation>
```
Comment (eumagga0x2a) [inline avidemux/qt4/i18n/avidemux_ko.ts]: Please do not correct strings which are wrong but don't need to be translated via translation. They need to be fixed in the application. The right thing to do would be to replace `<translation>© 2001 - 2017  Mean</translation>` with `<translation></translation>`.

Comment (sheppaul) [reply avidemux/qt4/i18n/avidemux_ko.ts]: If it should be fixed in the application, I have no idea the reason why the string is in translation text though I don't really care about it.

File: avidemux/qt4/i18n/avidemux_ko.ts (hunk 1 above)
Comment (sheppaul) [reply avidemux/qt4/i18n/avidemux_ko.ts]: See the attached screenshot. It is the only entry having a kbd accelerator. It just looks weird to me. I'll add it again someday if required. :p

File: avidemux/qt4/i18n/avidemux_ko.ts (hunk 2 above)
Comment (eumagga0x2a) [reply avidemux/qt4/i18n/avidemux_ko.ts]: The string is picked up automatically while parsing avidemux/qt4/ADM_userInterfaces/ADM_dialog/about.ui where we have a few strings which really need translation, so excluding the entire file is not an option.

Translation files will be regenerated shortly before release and changes to this string in the source will invalidate existing translations.

File: avidemux/qt4/i18n/avidemux_ko.ts (hunk 1 above)
Comment (eumagga0x2a) [reply avidemux/qt4/i18n/avidemux_ko.ts]: Well, all other actions except of 편집 초기화 (Reset Edit) are directly accessible via kbd shortcuts (on macOS, unlike other operating systems, preferences have the default kbd shortcut "Command + ," and are located in another menu) and preferences are a menu entry one uses quite often, methinks. There are accels in the Video menu too.

Anyway, I'll merge this as-is and leave accels to your discretion. Thank you for your contribution.

**Pull request merged**